d3b-center/fw-captk-whitestripe-normalization
# Permission_error_Bad_interpreter in bash script

Overview:

This pull request will address the Permission error / Bad interpreter issues of the bash (run) file script.

Connects #1

Checklist:
1. Dockerfile
2. Manifest.json
3. run - bash script 
 
Testing Instructions:

Make sure the file type of run and docker files are unix executable.
We can change it using chmod 755 filename.
